cmd/main.go

```diff
@@ -160,3 +181,25 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+// getWatchNamespace returns the Namespace the operator should be watching for changes
+func getWatchNamespace() (string, error) {
+	var watchNamespaceEnvVar = "WATCH_NAMESPACE"
+
+	ns := os.Getenv(watchNamespaceEnvVar)
+	if ns == "" {
+		return "", fmt.Errorf("%s must be set", watchNamespaceEnvVar)
+	}
+	return ns, nil
+}
+
+// getOperatorNamespace returns the Namespace the operator is running
+func getOperatorNamespace() (string, error) {
```

Comment:
we already have `OperatorNamespace` in utils. can you please use it?

Reply:
Could you pls point, didn't find it. There's something similar in controllers but not usable.

Reply:
sorry i was looking at my local copy where the getOperatorNamespace was exposed, can you please update https://github.com/ceph/ceph-csi-operator/blob/bdd5baa89f31c9f3d3ebfc17edcd3c7768931e55/internal/controller/defaults.go#L68 to use it outside it would duplicate functions

Reply:
as mentioned _in controllers but not usable_, that function serves different purpose, I can only move the env to const.

Reply:
I am trying to avoid duplication of the ENV var, so if we change something tomorrow, I don't need to change it in multiple places.

Reply:
done.